### ISSUE TITLE
Allow user to perform server actions

### ISSUE THREAD

**Issue by Wraithers** [gugahoi/maraschino#22]
Actions can be any of the following:
- Update libraries (all / section)
- _add more_

**Comment by gugahoi** [comment]
Refreshing libraries should be easy, what other kind of actions are you thinking about?
:32400/library/sections/2/refresh

**Comment by Wraithers** [comment]
You're probably using that as example to refresh just the TV Shows, which brings to mind an option for each of the sections to refresh individually, maybe through a single button at the bottom of the module, like Sickbeard's search.

However, is there also a way to refresh the entire library? Not just by section number? If not, then maybe you'd need to cycle through each of the commands or something.

I can't think of any other server actions right now that would be entirely necessary from Maraschino, but maybe I or someone else can add more over time.

**Comment by gugahoi** [comment]
Fair enough, I have added individual refresh buttons on each section, it is also possible to add a "refresh all" if wanted but I havent yet.

**Comment by Hellowlol** [comment]
I would like a update lib button for the entire server

**Comment by gugahoi** [comment]
@Hellowlol  that seems easy to do, just remind me if I haven't implemented it in a couple of weeks.
